crates/shared/src/schema_hash_approval.rs

```diff
@@ -8,7 +8,7 @@
 // If the schema in this library has changed then:
 //  1. Bump the version in `crates/shared/Cargo.toml`
 //  2. Change the `SCHEMA_VERSION` in this library to this new Cargo.toml version
-const APPROVED_SCHEMA_FILE_HASH: &str = "10197913343580353876";
```

Comment:
Needs another update.

Reply:
done in 9efa4f264b9cde6cb6f604c69eba57dde8f9facc